contentreview-impl/impl/src/java/org/sakaiproject/turnitin/util/TurnitinLTIUtil.java

```diff
@@ -87,6 +89,11 @@ public void init() {
 		if(StringUtils.isEmpty(turnitinSite)){
 			log.error("Turnitin global site property does not exist or is wrongly configured.");
 		}
+		turnitinReportsSite = serverConfigurationService.getString("turnitin.reports.lti.site", "!turnitin_reports");
```

Comment:
I developed this under the assumption that every LTI tool has 1 launch_url. So if we want to access both the /assignment and the /report endpoint, you'd need two instances. But now that I've added this 'launch suffix' idea, I wonder if we could set the launch_url to https://sandbox.turnitin.com/api/lti/1p0/dv/ and always pass "/assignment" or "/report" as an lti_launchsuffix.